lib/cppcheck.cpp

```diff
@@ -544,6 +544,13 @@ unsigned int CppCheck::checkClang(const std::string &path)
 
 unsigned int CppCheck::check(const std::string &path)
 {
+    if (!Path::isFile(path)) {
+        std::string fixedpath = Path::simplifyPath(path);
+        fixedpath = Path::toNativeSeparators(fixedpath);
+        const std::string errorMsg("File " + fixedpath + " does not exists. Skipping file.");
+        fileNotFoundError(Path::simplifyPath(path), errorMsg);
+        return 0;
```

Comment:
hmm if I remember correctly it would make sense to return a non-zero code.. do we return 0 after a internalError etc?